metricsmentor/main/views.py

```diff
@@ -225,9 +225,32 @@ def calculate_pvalue(request):
         p_value_two_sided = 2 * t.cdf(-abs(t_value), df)
 
         return JsonResponse({
-            'p_value_left': p_value_left,
-            'p_value_right': p_value_right,
-            'p_value_two_sided': p_value_two_sided
+            'value_left': p_value_left,
+            'value_right': p_value_right,
+            'value_two_sided': p_value_two_sided
+        })
+
+    return JsonResponse({'error': 'Invalid data or method.'}, status=400)
+
+
+@csrf_exempt
+def calculate_critical_value(request):
+    if request.method == 'POST':
+        data = json.loads(request.body.decode('utf-8'))
+        alpha = data.get('alpha')  # Significance level
+
+        # Calculate critical values
+        critical_value_left = -1.28 if alpha == 0.10 else (
```

Comment:
This is probably okay, but when comparing floats in python with `==`, I have heard that it's possible to have unexpected results. In general I've seen the absolute value for floats compared, when doing this sort of thing.

After some research, it looks like python 3.5 introduced the [math.isclose](https://docs.python.org/3/library/math.html#math.isclose) function, which may be more reliable for this comparison. 

```
critical_value_left = -1.28 if math.isclose(alpha, 0.10) else (
    -1.64 if math.isclose(alpha, 0.05) else -2.33)
```

* https://stackoverflow.com/questions/5595425/what-is-the-best-way-to-compare-floats-for-almost-equality-in-python
* https://docs.python.org/3/library/math.html#math.isclose

Reply:
Thanks Nik!